poleno/attachments/migrations/0003_content_type.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import magic
+
+from django.db import models, migrations
+
+def forward(apps, schema_editor):
+    Attachment = apps.get_model(u'attachments', u'Attachment')
+    mime = magic.Magic(mime=True)
+    for attachment in Attachment.objects.all():
+        content_type = mime.from_buffer(attachment.file.read())
+        attachment.content_type = content_type
```

Comment:
Ak pre nejaky subor uz v DB mame spravny content type, tak ho upravovat netreba. Pls pridaj podmienku, co porovna novy a stary content type a savne iba ak sa zmeni. Vid ako je to v `0002_name_encoding.py`.

Attachmentov mame desatisice a predpokladame, ze u drvivej veciny mame content type spravne.